vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story

```diff
@@ -14,7 +14,8 @@ Given I start mobile application with capabilities:
 |app |${app-url}|
 
 
-Scenario: Validate coordinate/size dynamic variables
+Scenario: Validate coordinate/size dynamic variables, page source dynamic variable
+Then `${app-source}` matches `.+Home.+`
```

Comment:
```suggestion
Then `${source-code}` matches `.+Home.+`
```

Reply:
I think it also makes sense to validate this variable for web applications as well